examples/gocean/eg5/profile/README.md

```diff
@@ -89,10 +89,10 @@ library, you should see:
  ...
  profile_PSyDataInit called
  ...
- PreStart called for module 'psy_test' region 'invoke_0:r0'
- PostEnd called for module 'psy_test' region 'invoke_0:r0'
- PreStart called for module 'psy_test' region 'invoke_1_update_field:update_field_code:r0'
- PostEnd called for module 'psy_test' region 'invoke_1_update_field:update_field_code:r0'
+ PreStart called for module 'invoke_0' region 'r0'
```

Comment:
I see what you mean about this. Really we should check the outputs of the examples as part of the testing but that's quite a lot of work I suspect.

Reply:
I've opened #2641. I am not aware that I changed anything here.